docs/tutorials/mutual_coupling_example.ipynb

```diff
@@ -118,7 +118,7 @@
   },
```

Comment:
<p>Add a reference to Rath &amp; Pascua et al. (2024): <a href="https://arxiv.org/abs/2406.08549" target="_blank">https://arxiv.org/abs/2406.08549</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_sim/pull/238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Good! Done</p>

Reply:
<p>Awesome, thanks!</p>